src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
The only place we use it--only one place in master--is with native events. It's an undocumented non-public API so our concern was just with internal use cases.